CMakeLists.txt

```diff
@@ -367,7 +367,11 @@ endif ()
 
 # libwasm, which implenents the wasm C API
 if (BUILD_LIBWASM)
-  add_library(wasm SHARED ${WABT_LIBRARY_SRC} src/interp/interp-wasm-c-api.cc)
+  if (WIN32)
+    add_library(wasm SHARED ${WABT_LIBRARY_SRC} src/interp/interp-wasm-c-api.cc)
+  else ()
+    add_library(wasm STATIC ${WABT_LIBRARY_SRC} src/interp/interp-wasm-c-api.cc)
```

Comment:
I'm not sure how cmake normally handles this but I would have thought this would be some kind of build option.   Most libraries build as both dynamic and static right?    We should try to follow cmake best practices whatever they are.